db/queries/food_item_log.sql

```diff
@@ -30,14 +31,80 @@ SELECT
     l.region,
     l.weight,
     l.weight_unit,
-    l.log_date
+    l.log_date,
+    l.carbon_footprint
 FROM
     food_item_log l
     INNER JOIN food_item f ON l.food_item_id = f.id
 WHERE
     user_id = $1
     AND log_date = $2;
 
+-- name: GetFoodItemLog :many
+SELECT
+    l.id,
+    f.name,
+    l.food_item_id,
+    l.region,
+    l.weight,
+    l.weight_unit,
+    l.carbon_footprint
+FROM
+    food_item_log l
+    INNER JOIN food_item f ON l.food_item_id = f.id
+WHERE
+    user_id = $1;
+
+-- name: GetDailyAverageCarbonFootprintByDietaryRequirement :one
+SELECT
+    average_carbon_footprint
+FROM
+    daily_user_average_by_dietary_requirement
+WHERE
+    dietary_requirement = $1
+LIMIT 1;
+
+-- name: GetDailyAverageCarbonFootprint :one
+SELECT
+    average_carbon_footprint
+FROM
+    daily_user_average
+LIMIT 1;
+
+-- name: GetFoodItemLogStreak :one
```

Comment:
If there is a longer streak before the current date, then that streak is picked as the max even if there was an active streak. I could create a separate query for the active streak.